analyzer/src/funTest/assets/projects/synthetic/spdx-project-expected-output.yml

```diff
@@ -39,7 +39,7 @@ packages:
     \ of powerful features."
   homepage_url: "https://curl.haxx.se/"
   binary_artifact:
-    url: "https://github.com/curl/curl/releases/download/curl-7_70_0/curl-7.70.0.tar.gz"
+    url: ""
```

Comment:
I'm confused why the content here changes. You're still referring to the same `package.spdx.yml` file for cURL, just from another location. Can you elaborate?

Reply:
Thanks for pointing that out. I realized I was using a link to an older version of the package.spdx.yml, which still had different metadata than the current one. I am now pointing to the raw content of the file in the master branch. 
There still will be a change for the vcs information, because it has no local directory it is pointing to anymore. I mentioned that in the commit message.